[Bug] [Connector-Starrocks] decimal column to wrong defination

### Search before asking

- [x] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

when schema evolution triggered; and  starrocks reviced AlterTableModifyColumnEvent or AlterTableModifyColumnEvent event， will call org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksTypeConverter#reconvert to generate a defination, but when the seatunnel type was DECIMAL, it ation like below:
``` java
builder.columnType(String.format("%s(%s,%s)", SR_DECIMAL, precision, scale));
builder.dataType(SR_DECIMALV3);
builder.precision((long) precision);
builder.scale(scale);
```
column type went wrong, it should be DECIMAL;
this cause this:
``` log
ErrorDescription:[Catalog initialize failed] - Failed connecting to jdbc:mysql://xxxx:9030/ via JDBC.
        at org.apache.seatunnel.engine.server.task.flow.SinkFlowLifeCycle.received(SinkFlowLifeCycle.java:302)
        at org.apache.seatunnel.engine.server.task.flow.SinkFlowLifeCycle.received(SinkFlowLifeCycle.java:70)
        at org.apache.seatunnel.engine.server.task.SeaTunnelTransformCollector.collect(SeaTunnelTransformCollector.java:39)
        at org.apache.seatunnel.engine.server.task.SeaTunnelTransformCollector.collect(SeaTunnelTransformCollector.java:27)
        at org.apache.seatunnel.engine.server.task.group.queue.IntermediateBlockingQueue.handleRecord(IntermediateBlockingQueue.java:75)
        at org.apache.seatunnel.engine.server.task.group.queue.IntermediateBlockingQueue.collect(IntermediateBlockingQueue.java:50)
        at org.apache.seatunnel.engine.server.task.flow.IntermediateQueueFlowLifeCycle.collect(IntermediateQueueFlowLifeCycle.java:51)
        at org.apache.seatunnel.engine.server.task.TransformSeaTunnelTask.collect(TransformSeaTunnelTask.java:72)
        at org.apache.seatunnel.engine.server.task.SeaTunnelTask.stateProcess(SeaTunnelTask.java:165)
        at org.apache.seatunnel.engine.server.task.TransformSeaTunnelTask.call(TransformSeaTunnelTask.java:77)
        at org.apache.seatunnel.engine.server.TaskExecutionService$BlockingWorker.run(TaskExecutionService.java:694)
        at org.apache.seatunnel.engine.server.TaskExecutionService$NamedTaskWrapper.run(TaskExecutionService.java:1023)
        at org.apache.seatunnel.api.tracing.MDCRunnable.run(MDCRunnable.java:43)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:750)
Caused by: org.apache.seatunnel.api.table.catalog.exception.CatalogException: ErrorCode:[API-03], ErrorDescription:[Catalog initialize failed] - Failed connecting to jdbc:mysql://xxxx9030/ via JDBC.
        at org.apache.seatunnel.connectors.seatunnel.starrocks.sink.StarRocksSinkWriter.applySchemaChange(StarRocksSinkWriter.java:101)
        at org.apache.seatunnel.api.sink.multitablesink.MultiTableSinkWriter.applySchemaChange(MultiTableSinkWriter.java:163)
        at org.apache.seatunnel.engine.server.task.flow.SinkFlowLifeCycle.received(SinkFlowLifeCycle.java:258)
        ... 17 more
Caused by: java.sql.SQLSyntaxErrorException: Getting syntax error at line 1, column 63. Detail message: No viable statement for input 'MODIFY COLUMN `xxxx` DECIMALV3'.
        at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
        at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
        at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
        at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
        at org.apache.seatunnel.connectors.seatunnel.starrocks.util.SchemaUtils.applySchemaChange(SchemaUtils.java:199)
        at org.apache.seatunnel.connectors.seatunnel.starrocks.util.SchemaUtils.applySchemaChange(SchemaUtils.java:86)
        at org.apache.seatunnel.connectors.seatunnel.starrocks.util.SchemaUtils.applySchemaChange(SchemaUtils.java:63)
        at org.apache.seatunnel.connectors.seatunnel.starrocks.sink.StarRocksSinkWriter.applySchemaChange(StarRocksSinkWriter.java:98)
```

### SeaTunnel Version

2.3.11

### SeaTunnel Config

```conf
no matter
```

### Running Command

```shell
no matter
```

### Error Exception

```log
in what happened
```

### Zeta or Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/datatypes/StarRocksType.java
+++ b/seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/datatypes/StarRocksType.java
@@ -32,7 +32,6 @@ public class StarRocksType {
     public static final String SR_FLOAT = "FLOAT";
     public static final String SR_DOUBLE = "DOUBLE";
     public static final String SR_DECIMAL = "DECIMAL";
-    public static final String SR_DECIMALV3 = "DECIMALV3";
     public static final String SR_DATE = "DATE";
     public static final String SR_DATETIME = "DATETIME";
     public static final String SR_CHAR = "CHAR";
@@ -46,22 +45,17 @@ public class StarRocksType {
     public static final String SR_BIGINT_ARRAY = "ARRAY<bigint>";
     public static final String SR_FLOAT_ARRAY = "ARRAY<float>";
     public static final String SR_DOUBLE_ARRAY = "ARRAY<double>";
-    public static final String SR_DECIMALV3_ARRAY = "ARRAY<DECIMALV3>";
-    public static final String SR_DECIMALV3_ARRAY_COLUMN_TYPE_TMP = "ARRAY<DECIMALV3(%s, %s)>";
-    public static final String SR_DATEV2_ARRAY = "ARRAY<DATEV2>";
-    public static final String SR_DATETIMEV2_ARRAY = "ARRAY<DATETIMEV2>";
+    public static final String SR_DECIMAL_ARRAY = "ARRAY<DECIMAL>";
+    public static final String SR_DECIMAL_ARRAY_COLUMN_TYPE_TMP = "ARRAY<DECIMAL(%s, %s)>";
+    public static final String SR_DATE_ARRAY = "ARRAY<DATE>";
+    public static final String SR_DATETIME_ARRAY = "ARRAY<DATETIME>";
     public static final String SR_STRING_ARRAY = "ARRAY<STRING>";
 
     // Because can not get the column length from array, So the following types of arrays cannot be
     // generated properly.
     public static final String SR_LARGEINT_ARRAY = "ARRAY<largeint>";
-    public static final String SR_CHAR_ARRAY = "ARRAY<CHAR>";
-    public static final String SR_CHAR_ARRAY_COLUMN_TYPE_TMP = "ARRAY<CHAR(%s)>";
-    public static final String SR_VARCHAR_ARRAY = "ARRAY<VARCHAR>";
-    public static final String SR_VARCHAR_ARRAY_COLUMN_TYPE_TMP = "ARRAY<VARCHAR(%s)>";
 
     public static final String SR_JSON = "JSON";
-    public static final String SR_JSONB = "JSONB";
 
     public static final String SR_ARRAY = "ARRAY";
 
@@ -71,10 +65,8 @@ public class StarRocksType {
     public static final String SR_ARRAY_INT_INTER = "int(11)";
     public static final String SR_ARRAY_BIGINT_INTER = "bigint(20)";
     public static final String SR_ARRAY_DECIMAL_PRE = "DECIMAL";
-    public static final String SR_ARRAY_DATE_INTER = "date";
-    public static final String SR_ARRAY_DATEV2_INTER = "DATEV2";
+    public static final String SR_ARRAY_DATE_INTER = "DATE";
     public static final String SR_ARRAY_DATETIME_INTER = "DATETIME";
-    public static final String SR_ARRAY_DATETIMEV2_INTER = "DATETIMEV2";
 
     public static final String SR_MAP = "MAP";
     public static final String SR_MAP_COLUMN_TYPE = "MAP<%s, %s>";

--- a/seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/datatypes/StarRocksTypeConverter.java
+++ b/seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/datatypes/StarRocksTypeConverter.java
@@ -42,9 +42,7 @@ import java.util.regex.Pattern;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_ARRAY;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_ARRAY_BIGINT_INTER;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_ARRAY_BOOLEAN_INTER;
-import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_ARRAY_DATETIMEV2_INTER;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_ARRAY_DATETIME_INTER;
-import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_ARRAY_DATEV2_INTER;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_ARRAY_DATE_INTER;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_ARRAY_DECIMAL_PRE;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_ARRAY_INT_INTER;
@@ -58,12 +56,11 @@ import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.Star
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_CHAR;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATE;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATETIME;
-import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATETIMEV2_ARRAY;
-import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATEV2_ARRAY;
+import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATETIME_ARRAY;
+import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATE_ARRAY;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DECIMAL;
-import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DECIMALV3;
-import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DECIMALV3_ARRAY;
-import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DECIMALV3_ARRAY_COLUMN_TYPE_TMP;
+import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DECIMAL_ARRAY;
+import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DECIMAL_ARRAY_COLUMN_TYPE_TMP;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DOUBLE;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DOUBLE_ARRAY;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_FLOAT;
@@ -91,7 +88,6 @@ public class StarRocksTypeConverter implements TypeConverter<BasicTypeDefine<Sta
     public static final long MAX_STRING_LENGTH = 2147483643;
     public static final Long MAX_PRECISION = 38L;
     public static final Integer MAX_SCALE = 10;
-    public static final Integer MAX_DATETIME_SCALE = 6;
     public static final long POWER_2_8 = (long) Math.pow(2, 8);
 
     public static final StarRocksTypeConverter INSTANCE = new StarRocksTypeConverter();
@@ -148,7 +144,6 @@ public class StarRocksTypeConverter implements TypeConverter<BasicTypeDefine<Sta
                 builder.dataType(BasicType.DOUBLE_TYPE);
                 break;
             case SR_DECIMAL:
-            case SR_DECIMALV3:
                 setDecimalType(builder, typeDefine);
                 break;
             case SR_CHAR:
@@ -290,8 +285,8 @@ public class StarRocksTypeConverter implements TypeConverter<BasicTypeDefine<Sta
                             precision,
                             scale);
                 }
-                builder.columnType(String.format("%s(%s,%s)", SR_DECIMALV3, precision, scale));
-                builder.dataType(SR_DECIMALV3);
+                builder.columnType(String.format("%s(%s,%s)", SR_DECIMAL, precision, scale));
+                builder.dataType(SR_DECIMAL);
                 builder.precision((long) precision);
                 builder.scale(scale);
                 break;
@@ -381,11 +376,9 @@ public class StarRocksTypeConverter implements TypeConverter<BasicTypeDefine<Sta
                     new DecimalArrayType(
                             new DecimalType(precisionAndScale[0], precisionAndScale[1]));
             builder.dataType(decimalArray);
-        } else if (columnInterType.equalsIgnoreCase(SR_ARRAY_DATE_INTER)
-                || columnInterType.equalsIgnoreCase(SR_ARRAY_DATEV2_INTER)) {
+        } else if (columnInterType.equalsIgnoreCase(SR_ARRAY_DATE_INTER)) {
             builder.dataType(ArrayType.LOCAL_DATE_ARRAY_TYPE);
-        } else if (columnInterType.equalsIgnoreCase(SR_ARRAY_DATETIME_INTER)
-                || columnInterType.equalsIgnoreCase(SR_ARRAY_DATETIMEV2_INTER)) {
+        } else if (columnInterType.equalsIgnoreCase(SR_ARRAY_DATETIME_INTER)) {
             builder.dataType(ArrayType.LOCAL_DATE_TIME_ARRAY_TYPE);
         } else if (columnInterType.equalsIgnoreCase(SR_LARGEINT)) {
             DecimalArrayType decimalArray = new DecimalArrayType(new DecimalType(20, 0));
@@ -539,10 +532,10 @@ public class StarRocksTypeConverter implements TypeConverter<BasicTypeDefine<Sta
                 int[] precisionAndScale = getPrecisionAndScale(elementType.toString());
                 builder.columnType(
                         String.format(
-                                SR_DECIMALV3_ARRAY_COLUMN_TYPE_TMP,
+                                SR_DECIMAL_ARRAY_COLUMN_TYPE_TMP,
                                 precisionAndScale[0],
                                 precisionAndScale[1]));
-                builder.dataType(SR_DECIMALV3_ARRAY);
+                builder.dataType(SR_DECIMAL_ARRAY);
                 break;
             case STRING:
             case TIME:
@@ -550,12 +543,12 @@ public class StarRocksTypeConverter implements TypeConverter<BasicTypeDefine<Sta
                 builder.dataType(SR_STRING_ARRAY);
                 break;
             case DATE:
-                builder.columnType(SR_DATEV2_ARRAY);
-                builder.dataType(SR_DATEV2_ARRAY);
+                builder.columnType(SR_DATE_ARRAY);
+                builder.dataType(SR_DATE_ARRAY);
                 break;
             case TIMESTAMP:
-                builder.columnType(SR_DATETIMEV2_ARRAY);
-                builder.dataType(SR_DATETIMEV2_ARRAY);
+                builder.columnType(SR_DATETIME_ARRAY);
+                builder.dataType(SR_DATETIME_ARRAY);
                 break;
             default:
                 throw CommonError.convertToConnectorTypeError(

--- a/seatunnel-connectors-v2/connector-starrocks/src/test/java/org/apache/seatunnel/connectors/seatunnel/starrocks/catalog/StarRocksTypeConverterTest.java
+++ b/seatunnel-connectors-v2/connector-starrocks/src/test/java/org/apache/seatunnel/connectors/seatunnel/starrocks/catalog/StarRocksTypeConverterTest.java
@@ -43,9 +43,9 @@ import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.Star
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_CHAR;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATE;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATETIME;
-import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATETIMEV2_ARRAY;
-import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATEV2_ARRAY;
-import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DECIMALV3;
+import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATETIME_ARRAY;
+import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DATE_ARRAY;
+import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DECIMAL;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DOUBLE;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_DOUBLE_ARRAY;
 import static org.apache.seatunnel.connectors.seatunnel.starrocks.datatypes.StarRocksType.SR_FLOAT;
@@ -255,7 +255,7 @@ public class StarRocksTypeConverterTest {
         BasicTypeDefine<StarRocksType> typeDefine =
                 BasicTypeDefine.<StarRocksType>builder()
                         .name("test")
-                        .columnType("decimalv3")
+                        .columnType("decimal")
                         .dataType("decimal")
                         .precision(9L)
                         .scale(2)
@@ -270,7 +270,7 @@ public class StarRocksTypeConverterTest {
         typeDefine =
                 BasicTypeDefine.<StarRocksType>builder()
                         .name("test")
-                        .columnType("decimalv3(36,2)")
+                        .columnType("decimal(36,2)")
                         .dataType("decimal")
                         .precision(38L)
                         .scale(2)
@@ -465,7 +465,7 @@ public class StarRocksTypeConverterTest {
         typeDefine =
                 BasicTypeDefine.<StarRocksType>builder()
                         .name("test")
-                        .columnType("array<decimalv3(10, 2)>")
+                        .columnType("array<decimal(10, 2)>")
                         .dataType("ARRAY")
                         .build();
         column = converter.convert(typeDefine);
@@ -598,7 +598,7 @@ public class StarRocksTypeConverterTest {
         typeDefine =
                 BasicTypeDefine.<StarRocksType>builder()
                         .name("test")
-                        .columnType("map<string,decimalv3(10, 2)>")
+                        .columnType("map<string,decimal(10, 2)>")
                         .dataType("MAP")
                         .build();
         column = converter.convert(typeDefine);
@@ -610,7 +610,7 @@ public class StarRocksTypeConverterTest {
         typeDefine =
                 BasicTypeDefine.<StarRocksType>builder()
                         .name("test")
-                        .columnType("map<decimalv3(10, 2),date>")
+                        .columnType("map<decimal(10, 2),date>")
                         .dataType("MAP")
                         .build();
         column = converter.convert(typeDefine);
@@ -779,19 +779,19 @@ public class StarRocksTypeConverterTest {
         Assertions.assertEquals(
                 String.format(
                         "%s(%s,%s)",
-                        SR_DECIMALV3,
+                        SR_DECIMAL,
                         StarRocksTypeConverter.MAX_PRECISION,
                         StarRocksTypeConverter.MAX_SCALE),
                 typeDefine.getColumnType());
-        Assertions.assertEquals(SR_DECIMALV3, typeDefine.getDataType());
+        Assertions.assertEquals(SR_DECIMAL, typeDefine.getDataType());
 
         column = PhysicalColumn.builder().name("test").dataType(new DecimalType(10, 2)).build();
 
         typeDefine = converter.reconvert(column);
         Assertions.assertEquals(column.getName(), typeDefine.getName());
-        Assertions.assertEquals(SR_DECIMALV3, typeDefine.getDataType());
+        Assertions.assertEquals(SR_DECIMAL, typeDefine.getDataType());
         Assertions.assertEquals(
-                String.format("%s(%s,%s)", SR_DECIMALV3, 10, 2), typeDefine.getColumnType());
+                String.format("%s(%s,%s)", SR_DECIMAL, 10, 2), typeDefine.getColumnType());
 
         column = PhysicalColumn.builder().name("test").dataType(new DecimalType(40, 2)).build();
 
@@ -1098,8 +1098,8 @@ public class StarRocksTypeConverterTest {
 
         typeDefine = converter.reconvert(column);
         Assertions.assertEquals(column.getName(), typeDefine.getName());
-        Assertions.assertEquals(SR_DATEV2_ARRAY, typeDefine.getColumnType());
-        Assertions.assertEquals(SR_DATEV2_ARRAY, typeDefine.getDataType());
+        Assertions.assertEquals(SR_DATE_ARRAY, typeDefine.getColumnType());
+        Assertions.assertEquals(SR_DATE_ARRAY, typeDefine.getDataType());
 
         column =
                 PhysicalColumn.builder()
@@ -1109,16 +1109,16 @@ public class StarRocksTypeConverterTest {
 
         typeDefine = converter.reconvert(column);
         Assertions.assertEquals(column.getName(), typeDefine.getName());
-        Assertions.assertEquals(SR_DATETIMEV2_ARRAY, typeDefine.getColumnType());
-        Assertions.assertEquals(SR_DATETIMEV2_ARRAY, typeDefine.getDataType());
+        Assertions.assertEquals(SR_DATETIME_ARRAY, typeDefine.getColumnType());
+        Assertions.assertEquals(SR_DATETIME_ARRAY, typeDefine.getDataType());
 
         DecimalArrayType decimalArrayType = new DecimalArrayType(new DecimalType(10, 2));
         column = PhysicalColumn.builder().name("test").dataType(decimalArrayType).build();
 
         typeDefine = converter.reconvert(column);
         Assertions.assertEquals(column.getName(), typeDefine.getName());
-        Assertions.assertEquals("ARRAY<DECIMALV3(10, 2)>", typeDefine.getColumnType());
-        Assertions.assertEquals("ARRAY<DECIMALV3>", typeDefine.getDataType());
+        Assertions.assertEquals("ARRAY<DECIMAL(10, 2)>", typeDefine.getColumnType());
+        Assertions.assertEquals("ARRAY<DECIMAL>", typeDefine.getDataType());
 
         decimalArrayType = new DecimalArrayType(new DecimalType(20, 0));
         column =
@@ -1129,8 +1129,8 @@ public class StarRocksTypeConverterTest {
                         .build();
         typeDefine = converter.reconvert(column);
         Assertions.assertEquals(column.getName(), typeDefine.getName());
-        Assertions.assertEquals("ARRAY<DECIMALV3(20, 0)>", typeDefine.getColumnType());
-        Assertions.assertEquals("ARRAY<DECIMALV3>", typeDefine.getDataType());
+        Assertions.assertEquals("ARRAY<DECIMAL(20, 0)>", typeDefine.getColumnType());
+        Assertions.assertEquals("ARRAY<DECIMAL>", typeDefine.getDataType());
     }
 
     @Test
@@ -1222,8 +1222,8 @@ public class StarRocksTypeConverterTest {
 
         typeDefine = converter.reconvert(column);
         Assertions.assertEquals(column.getName(), typeDefine.getName());
-        Assertions.assertEquals("MAP<DECIMALV3(10,2), STRING>", typeDefine.getColumnType());
-        Assertions.assertEquals("MAP<DECIMALV3(10,2), STRING>", typeDefine.getDataType());
+        Assertions.assertEquals("MAP<DECIMAL(10,2), STRING>", typeDefine.getColumnType());
+        Assertions.assertEquals("MAP<DECIMAL(10,2), STRING>", typeDefine.getDataType());
 
         column =
                 PhysicalColumn.builder()

